systers_portal/templates/meetup/members.html

```diff
@@ -3,7 +3,7 @@
 {% block meetup_location_page_content %}
   <div class="mb40">
     {% include "meetup/snippets/users-grid.html" with users="Organizers" user_list=object.organizers.all %}
-    {% include "meetup/snippets/users-grid.html" with users="Members" user_list=object.members.all %}
-  </div>
+    {% include "meetup/snippets/users-grid.html" with users="Members" user_list=object.members.all non_user_list=object.organizers.all %}
```

Comment:
An idea and general good practice in Django: let's move those database queries to the view. Making too many queries or any queries at all inside templates can lead to hard to debug code (someone might spend a fair amount of time scratching their head to figure out how the list of organizers is fetched).

Specifically since what you want to display is all members that are not organizers (in case someone is both an organizer and a member), we can use a more efficient way - a projection query and an exclude. It's a bit more complex than most of the ORM code in this project, so take a look at this [example](http://stackoverflow.com/a/22266776).

This will also heavily simplify the template code and revert back to passing just a simple `user_list` queryset.